### ISSUE TITLE
update-snapshots fails (3.0) due to missing jupyterlab setup

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/extension-cookiecutter-ts#258]
```
  set -eux
  jlpm
  python -m pip install .
  shell: /usr/bin/bash -e {0}
+ jlpm
/home/runner/work/_temp/883e7265-4e7f-4365-b0ea-df9e52ade86c.sh: line 2: jlpm: command not found
Error: Process completed with exit code 127.
```

Example https://github.com/jupyterlab/ui-profiler/actions/runs/3722798618/jobs/6313896572#logs

Ideally we would test actions.

**Comment by krassowski** [comment]
And then it fails with:

```
yarn run v1.22.19
error Command "test:update" not found.
```

**Comment by fcollonval** [comment]
Closed by #259
